website/source/docs/providers/aws/r/alb.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "aws"
+page_title: "AWS: aws_alb"
+sidebar_current: "docs-aws-resource-alb"
+description: |-
+  Provides an Application Load Balancer resource.
+---
+
+# aws\_alb
+
+Provides an Application Load Balancer resource.
+
+## Example Usage
+
+```
+# Create a new load balancer
+resource "aws_alb" "test" {
+  name            = "test-alb-tf"
+  internal        = false
+  security_groups = ["${aws_security_group.alb_sg.id}"]
+  subnets         = ["${aws_subnet.public.*.id}"]
+
+  disable_api_protection = true
```

Comment:
Where did this field come from? I was unable to find it anywhere in the whole diff.